# CircleInfo.msg
Header header    # 时间戳，单位为纳秒
int64 x
int64 y
int64 radius
int64 depth

int64 number
int64 type